roles/benchmarking/benchmarking_run_mlperf_ssd/tasks/main.yml

```diff
@@ -7,24 +7,24 @@
   command: oc get nodes -l kubernetes.io/hostname={{ benchmarking_run_mlperf_ssd_node_hostname }} -oname
 
 - name: Ensure that the coco dataset PVC exists
-  command:
-    oc get pvc/{{ benchmarking_run_mlperf_ssd_dataset_pvc_name }}
+  shell: |
+    oc get pvc/{{ benchmarking_run_mlperf_ssd_dataset_pvc_name }} \
        -n {{ benchmarking_run_mlperf_ssd_namespace }}
 
 - name: Fetch the coco dataset PVC definition (debug)
-  shell:
-    oc get pvc/{{ benchmarking_run_mlperf_ssd_pvc_name }}
-       -n {{ benchmarking_run_mlperf_ssd_namespace }}
-       -oyaml
+  shell: |
+    oc get pvc/{{ benchmarking_run_mlperf_ssd_pvc_name }} \
+       -n {{ benchmarking_run_mlperf_ssd_namespace }} \
+       -oyaml \
```

Comment:
I'm not sure to understand why you add the pipe in `shell: |`
without the pipe, the block is considered as a single line, so the trailing `\` isn't necessary, that makes the block easier to read from my POV
... but at the same time it may be confusing when you're not 100% clear about the semantic 🙃

Reply:
Yeah, multiline blocks with the pipe is how bash interprets code blocks by default, I have seen inconsistencies when using special chars in this type of code blocks.

For instance, this specific example breaks the syntax highlight in my code editor https://imgur.com/a/WFaenpt